Zotero RDF.js

```diff
@@ -518,6 +518,16 @@ function generateItem(item, zoteroType, resource) {
 	if (displayTitle) Zotero.RDF.addStatement(resource, n.z+"displayTitle", displayTitle, true);
 }
 
+function makeValidFileURI(path) {
+	path = path.replace(/#/g, '%23').replace(/\?/g, '#3F');
+	if (path.startsWith('/')) {
+		return 'file://' + path;
```

Comment:
[Apparently](https://en.wikipedia.org/wiki/File_URI_scheme#Windows_2) on Windows a valid path would be `file:///c:/path/to/the%20file.txt`, with three slashes. We'd also presumably need to replace backslashes with slashes. And maybe all the other things?

Did you test that we actually import file URLs properly? And was my guess right that we handle absolute paths now as a hack? I guess I'd worry that by making these neither valid file URLs nor paths, we'll break importing altogether, because it will throw an error trying to parse it as a URL but then not be a valid path either.

We'd want to add some tests for this afterwards, at least — we have some RDF tests in importExportTest.js in zotero/zotero already.